pkg/apis/extensions/types.go

```diff
@@ -676,6 +676,12 @@ type PodSecurityPolicySpec struct {
 	// If set to false the container may run with a read only root file system if it wishes but it
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty"`
+	// SeccompProfiles lists the allowed profiles that may be set for the pod or
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
```

Comment:
It's a bit strange here as seccomp only reduces permissions for a process. Not specifying means `unconfined` in the current implementation, i.e. not allowing anything means that the user is not allowed to restrict syscall permissions for his process.

I have read in the Docker 1.12 release notes that they switched to a dynamic default seccomp profile. It depends on the capabilities for a container. The seccomp profile "follows" the capabilities.

In kube we have implemented `unconfined` as default which is worse security-wise than docker's default. Maybe we should rethink that choice.